offload/DeviceRTL/src/Misc.cpp

```diff
@@ -128,6 +129,33 @@ double omp_get_wtime(void) { return ompx::impl::getWTime(); }
 void *__llvm_omp_indirect_call_lookup(void *HstPtr) {
   return ompx::impl::indirectCallLookup(HstPtr);
 }
+
+void *omp_alloc(size_t size, omp_allocator_handle_t allocator) {
+  switch (allocator) {
+  case omp_default_mem_alloc:
+  case omp_large_cap_mem_alloc:
```

Comment:
I think we need folks from the committee to clarify what the expected behavior would be on the device from the spec's perspective. @mjklemm @dreachem

Reply:
It's fine to map the default memory and the other types (like large capacity, HBM, etc) to the same memory space on a GPU.  But it needs to be document, because that behavior is implementation-defined and thus needs to be stated in the LLVM docs.

Reply:
Updated